client/src/components/History/ContentItem/Dataset/DatasetMenu.vue

```diff
@@ -273,6 +273,17 @@ export default {
             }
         },
 
+        showDetails() {
+            const redirectParams = {
+                path: this.dataset.getUrl("show_params"),
```

Comment:
Can you build the url directly ? I think we shouldn't stuff this in the dataset model.

Reply:
Meh, ignore that. We should do it, but don't need to do it now.